### PR TITLE
[multibarchart] when the interactive layer is on, displays highlights

### PR DESCRIPTION
**Pull request by bobylito**
When using the interactive layer on multibarchart, there was no highlight of the hovered series. This behavior is inconsistent with the one from the stackedAreaChart and the lineChart.

I added a sample that let you test the behaviour. Unfortunately I was unable to implement tests for this new behavior. Could anyone give me a hand on that?

### CODE REVIEW

**Comment by bobylito** [comment]
Can I help make this PR go through?

File: src/models/multiBarChart.js [hunk 1]
```diff
@@ -373,6 +373,18 @@ nv.models.multiBarChart = function() {
...
 
+                    //Highlight the tooltip entry based on which bar(s) the mouse is hovering.
+                    if (allData.length >= 2 && multibar.stacked()) {
+                        var yValue = chart.yScale().invert(e.mouseY);
+                        allData.forEach(function(d, currentIdx) {
+                            var isStacked = !!d.data.y1;
+                            if(isStacked && yValue >= d.data.y0 && yValue < d.data.y1) 
+                                allData[currentIdx].highlight = true;
+                            else if(!isStacked && yValue >= 0 && yValue < d.data.y)
+                                allData[currentIdx].highlight = true;
```
Comment (liquidpele) [inline src/models/multiBarChart.js]: please use brackets around all if/else blocks.

File: examples/multiBarChart3.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    .transition().duration(0)
+                    .call(graph);
+
+            });
+        }
+    });
+
+</script>
+</body>
+</html>
```
Comment (liquidpele) [inline examples/multiBarChart3.html]: Is a whole other example really necessary?  Can this feature not be shown on an existing example?

**Comment by liquidpele** [comment]
This should really be done to multiBarHorizontalChart too so that they match.

**Comment by bobylito** [comment]
Thanks @liquidpele for taking the time to have a look at this PR. Will make the necessary changes.

**Comment by liquidpele** [comment]
ping...

**Comment by liquidpele** [comment]
Closing this PR as abandoned...

**Comment by bobylito** [comment]
Sorry @liquidpele bad timing for me :(

**Comment by liquidpele** [comment]
No worries, par for the course with open source.   Hey that rhymes!

**Pull request closed without merging**